prettyprinter/src/Data/Text/Prettyprint/Doc/Internal.hs

```diff
@@ -1452,12 +1452,12 @@ fuse depth = go
 data SimpleDocStream ann =
       SFail
     | SEmpty
-    | SChar Char (SimpleDocStream ann)
+    | SChar !Char (SimpleDocStream ann)
 
     -- | Some layout algorithms use the Since the frequently used 'T.length' of
```

Comment:
TODO: Fix docs here.

Reply:
Done in https://github.com/quchen/prettyprinter/commit/aa3689515cc539bf355390d7d9754c470bb4dbcc.